css-grid-classes/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS Grid Classes</title>
+</head>
+<body>
+    <div class="container">
+      <div class="row">
+        <h1 class="column_full">Choose your character!</h1>
+      </div>
+
+      <div class="row">
+        <img class="column_half" src="./images/mario.png" alt="">
```

Comment:
make sure to wrap these images and all other images for this exercise in a column container. it is a best practice to put the column classes on the container, and then to set the width of the images in the containers to be 100% of the width of the containers. You don't want to put the column class directly on the img element.

Reply:
Aye aye, Captain! Here is the edited version based on your feedback.

![image](https://user-images.githubusercontent.com/73314663/99139599-81a58880-263a-11eb-9f69-bf35a452781c.png)
![image](https://user-images.githubusercontent.com/73314663/99139609-98e47600-263a-11eb-869d-66d4ef193055.png)